miscellaneous-commands/userconfig.md

```diff
@@ -1,6 +1,6 @@
 # userconfig
 
-<TODO: describe>
+This command lets you configure some personal settings regarding the types of notifications you get from MoonlightBot, and other prefrences regarding your experince with MoonlightBot.
```

Comment:
```suggestion
This command lets you configure some personal settings regarding the types of notifications you get from MoonlightBot, and other preferences to customize your experience with the bot.
```

I'd also mention that it works the same way as the /config (along with link) for servers